e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"flag"
+	"github.com/cucumber/godog"
+	"github.com/cucumber/godog/colors"
+	"os"
+	"testing"
+)
+
+var opts = godog.Options{
+	Output: colors.Colored(os.Stdout),
+	Format: "progress",
+}
+
+func init() {
+	godog.BindCommandLineFlags("godog.", &opts)
+}
+
+func InitializeIngestManagerTestSuite(ctx *godog.TestSuiteContext) {
+	initializeIngestManagerTestSuite(ctx)
+}
+
+func InitializeIngestManagerTestScenario(ctx *godog.ScenarioContext) {
+	initializeIngestManagerTestScenario(ctx)
+}
+
+func TestMain(m *testing.M) {
```

Comment:
Although I like this approach, I think it's mixing the unit tests from the godog tests.

There is a unit test layer (`make -C e2e unit-test`) that executes only unit tests, not BDD/godog tests, and it seems this change is adding the godog tests to the execution of the unit tests, which uses `[gotestsum](https://github.com/gotestyourself/gotestsum)` as test runner (uses go test plus pigments)